.bazelrc

```diff
@@ -1,16 +1,39 @@
-build --incompatible_enable_cc_toolchain_resolution
-build --action_env BAZEL_DO_NOT_DETECT_CPP_TOOLCHAIN=1
+# share bazel cache between checkouts of the same project
+# and keep old build caches around for longer
+common --disk_cache=~/.cache/shared_bazel_action_cache
+common --repository_cache=~/.cache/shared_bazel_repository_cache
+
+# better caching / reproducibility
+common --incompatible_strict_action_env=true
```

Comment:
Not sure if you know (it's a draft after all), but this line seems to break running pretty much anything except build? Or at least `bazel query` or really just `bazel version` is broken.

Reply:
Thanks for the report. I'll have a look tomorrow.
Wondering if this is a macOS thing but it may be an oversight from my side.

Reply:
You are absolutely right! Fixing this.